### DISCUSSION TITLE
Svg doesn't display in publish version

### DISCUSSION THREAD

**Discussion by limeniye** [unoplatform/uno#16672] [Q&A] [answered]
### Input information

I have an image.

```XML
<Image Height="50"
       HorizontalAlignment="Center"
       Source="ms-appx:///UnoApp77/Assets/Svg/backkkkk.svg"
       Stretch="Uniform" />
```

After executing the following command, 2 directories appear.

```PS
msbuild /r /t:Publish /p:Configuration=Release /p:RuntimeIdentifier=win-x64 /p:Platform=x64 /p:TargetFramework=net8.0-windows10.0.19041 /bl
```

But for some reason the image in the ```Release\net8.0-windows10.0.19041\win-x64\publish``` directory is not displayed.

![image](https://github.com/unoplatform/uno/assets/69314237/75a4b595-74b6-4853-8f3a-9820cf65819b)

### Output

[UnoApp77ImagesDoesntDisplayInPublish.zip](https://github.com/unoplatform/uno/files/15273213/UnoApp77ImagesDoesntDisplayInPublish.zip)

[msbuild.zip](https://github.com/unoplatform/uno/files/15273196/msbuild.zip)

### Package versions

```XML
<PackageVersion Include="Uno.Resizetizer" Version="1.5.0-dev.58" />
<PackageVersion Include="Uno.WinUI" Version="5.2.95" />
```

### Platforms

- [x] WinUI

**Comment by jeromelaban** [comment]
This looks like WinAppSDK issue, as I can repro the same behavior with a bare WinAppSDK app. Try moving your app to single project to avoid having assets in a separate library.

**Reply by jeromelaban** [reply, marked as answer]
Tracked in https://github.com/unoplatform/uno/issues/16674

**Reply by limeniye** [reply]
_offtop_
>Try moving your app to single project

@jeromelaban, I'll try it as soon as possible. Currently, I'm too busy with tasks
